### PR TITLE
Create Rule5-33.md

### PR DESCRIPTION
**Pull request by xingcx**

### CODE REVIEW

File: docs/section5/Rule5-33.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+**Applicability Checks:** None  
+
+**Manual Checks:** None  
+**Evaluation Context:**  Each Data Element  
+**Data Lookup:** None  
+**Function Call:**  
+
+  1. match_data_element()
+  2. get_opaque_surface_type()
+
```
Comment (mkarpman) [inline docs/section5/Rule5-33.md:20]: I don't think we need this function

File: docs/section5/Rule5-33.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- For each building segment in the Proposed model: `for building_segment_p in P_RMR.building.building_segments:`
+
+  - For each thermal block in building segment: `for thermal_block_p in building_segment_p.thermal_blocks:`
+
+    - For each zone in thermal block: `for zone_p in thermal_block_p.zones:`
+
+      - For each surface in zone: `for surface_p in zone_p.surfaces:`
+
+        - Check if surface is above-grade wall: `if get_opaque_surface_type(surface_p) == "ABOVE-GRADE WALL":`
+
```
Comment (mkarpman) [inline docs/section5/Rule5-33.md:32]: Don't need this check

Comment (supriyagoel) [reply docs/section5/Rule5-33.md:32]: P RMR should have same number of sub-surfaces as U-RMR

File: docs/section5/Rule5-33.md (hunk 1 above)
Comment (xingcx) [reply docs/section5/Rule5-33.md:20]: Wall check and the function is now removed.

File: docs/section5/Rule5-33.md (hunk 2 above)
Comment (xingcx) [reply docs/section5/Rule5-33.md:32]: Wall check and the function is now removed.

**Review by mkarpman — APPROVED** [review]

**Review by supriyagoel — APPROVED** [review]

File: docs/section5/Rule5-33.md [hunk 3]
```diff
@@ -28,16 +27,14 @@
...
+        - For each subsurface in surface: `for subsurface_p in surface_p.subsurfaces:`
 
-          - For each subsurface in surface: `for subsurface_p in surface_p:`
+          - Get matching subsurface in U_RMR: `subsurface_u = match_data_element(U_RMR, Subsurfaces, subsurface_p.id)`
 
-            - Get matching subsurface in U_RMR: `subsurface_u = match_data_element(U_RMR, Subsurfaces, subsurface_p.id)`
+            **Rule Assertion:**
 
-              **Rule Assertion:**
+            - Case 1: For each vertical fenestration in P_RMR, if automatically controlled shading devices are modeled the same as in U_RMR: `if subsurface_p.has_automatic_shades == subsurface_u.has_automatic_shades: PASS`
```
Comment (mkarpman) [inline docs/section5/Rule5-33.md:37]: Please replace "For each vertical fenestration in P_RMR ... " with "For each subsurface in P_RMR ... "

**Review by mkarpman — APPROVED** [review]

File: docs/section5/Rule5-33.md (hunk 3 above)
Comment (xingcx) [reply docs/section5/Rule5-33.md:37]: Corrected.

**Review by mkarpman — APPROVED** [review]

**Pull request merged**